### PR TITLE
Added object counting feature

### PR DESCRIPTION
**Pull request by Erol444**
`python3.6 depthai_demo.py --count 5`
image

`python3.6 depthai_demo.py --count tvmonitor`
image

I'm using bottles as an example because the model detects them accurately. 
Any ideas on how to get multiples of the object? So it would display eg. `2 people` instead of `2 person`

### CODE REVIEW

**Comment by Luxonis-Brandon** [comment]
Looks great!

File: depthai_demo.py [hunk 1]
```diff
@@ -226,6 +226,9 @@ def __init__(self, source, model_dir=None, model_name=None):
         if conf.args.cnn_input_size:
             self.input_size = tuple(map(int, conf.args.cnn_input_size.split('x')))
 
+        # Count objects detected on the frame
+        self.count = conf.getCountLabel(self)
```
Comment (VanDavv) [inline depthai_demo.py:230]: Can we rename the self.count and conf.args.count to `count_label`? 

The cli param can stay the same, adding `dest` flag to argparse should make it work, it's only a matter of knowing that does a count variable mean

File: depthai_demo.py [hunk 2]
```diff
@@ -379,6 +382,21 @@ def draw_detection(frame, detection):
...
                 else:
                     draw_detection(source, detection)
+
+            if self.count is not None:
+                def draw_cnt(frame, cnt):
+                    cv2.rectangle(frame, (0, 35), (120, 50), (255, 255, 255), cv2.FILLED)
+                    cv2.putText(frame, f"{self.count}: {cnt}", (5, 46), fps_type, 0.5, fps_color)
+
+                # Count the number of detected objects
+                cnt_iter = filter(lambda x: self.get_label_text(x.label) == self.count, decoded_data)
```
Comment (VanDavv) [inline depthai_demo.py:392]:
```suggestion
                cnt_iter = list(filter(lambda x: self.get_label_text(x.label) == self.count, decoded_data))
```

File: depthai_demo.py [hunk 3]
```diff
@@ -379,6 +382,21 @@ def draw_detection(frame, detection):
...
+                cnt = sum(1 for _ in cnt_iter)
```
Comment (VanDavv) [inline depthai_demo.py:393]:
```suggestion
                cnt = len(cnt_iter)
```

File: depthai_demo.py (hunk 2 above)
Comment (VanDavv) [reply depthai_demo.py:392]: And it would also change it's name from `cnt_iter` to `cnt_list` or other

File: depthai_helpers/arg_manager.py [hunk 4]
```diff
@@ -84,4 +84,5 @@ def parse_args():
                         help="Mono cam fps: max 60.0 for H:720 or H:800, max 120.0 for H:400. Default: %(default)s")
     parser.add_argument('-cb', '--callback', type=Path, default=project_root / 'callbacks.py', help="Path to callbacks file to be used. Default: %(default)s")
     parser.add_argument("--openvino_version", type=str, choices=openvino_versions, help="Specify which OpenVINO version to use in the pipeline")
+    parser.add_argument("--count", type=str, help="Count and display the number of specified objects on the frame. You have to specify the object (label) you want to count.")
```
Comment (VanDavv) [inline depthai_helpers/arg_manager.py:87]: I would add that the label can be either a label id or label name (if defined labels in nn config file)

**Review by VanDavv — CHANGES_REQUESTED** [review]

**Review by VanDavv — APPROVED** [review]

**Pull request merged**